cami.c

```diff
@@ -101,14 +101,17 @@ struct ami_session {
 	unsigned int return_null_on_error:1;
 };
 
+static int ami_debug_level = 0;
```

Comment:
Add a comment that this is simply the debug level for session creation in `ami_connect`, rather than the actual debug level of any particular session afterwards or the library as a whole (which might be assumed). Could even rename to `ami_initial_debug_level` or something for clarity.